### ISSUE TITLE
Make scrollbars squeeze on overdrag

### ISSUE THREAD

**Issue by SpyrexDE** [SpyrexDE/SmoothScroll#19]
Can be seen on this site: https://idiotwu.github.io/smooth-scrollbar/

**Comment by SpyrexDE** [comment]
I already tried some things but I could not find a good solution without too many workarounds. Maybe you find something, @HaroldLever?

**Comment by HaroldLever** [comment]
Hi, @SpyrexDE . So

**Comment by HaroldLever** [comment]
@SpyrexDE Sorry for replying late, busy about Game Jam these days.

I think you can get_h/v_scrollbar and change its Page value to makes it looks like squeezing.

**Comment by SpyrexDE** [comment]
Yeah, I already had to experience that my self when experimenting. The only way I see to achieve this effect using Godot's scroll bar is by hiding the real scroll bar and creating a new fake scrollcontainer above the real one. Then the position and size of the bar gets synchronized and when there is some overdrag, the size of the overlaying scroll container gets enlarged which results in a smaller scroll bar.
It's a workaround but still uses Godot's scroll bars so I think it's the best solution. What do you think?

**Comment by HaroldLever** [comment]
That sounds practicable, it worths a try. Looking forwad the final effect.
